duckduckgo/iOS
# Shared DB handling for CoreData storage

Task/Issue URL:  https://app.asana.com/0/1201493110486074/1203021259896122
Tech Design URL:
CC:

**Description**:

Shared DB handling for CoreData storage



**Steps to test this PR**:

Ensure DB works as expected in regular usage.
Check migration and clean install.



**Device Testing**:


**OS Testing**:



---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
